packages/@aws-cdk/aws-servicecatalog/test/integ.product.ts

```diff
@@ -21,29 +21,35 @@ const productStackHistory = new ProductStackHistory(stack, 'ProductStackHistory'
   currentVersionLocked: true,
 });
 
+const locallySavedVersions = productStackHistory.allVersions();
```

Comment:
This test is getting quite abstract. I think it's time to add some assertions to it. 

See [INTEGRATION_TESTS.md](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md) and [integ-tests](https://docs.aws.amazon.com/cdk/api/v2/docs/integ-tests-alpha-readme.html).